lms/static/sass/inline-discussion.scss

```diff
@@ -0,0 +1,33 @@
+// From lms-main-v1.scss
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
```

Comment:
@e-kolpakov I created a corresponding file for RTL, and updated `DiscussionXBlock` to conditionally load it (based on output of `get_language_bidi`).

I didn't add an `-ltr` suffix to the original file though: The naming convention for files in `lms/static/sass` is to add a suffix for RTL stylesheets, and to leave it off for LTR stylesheets (`lms-main-v1.scss` vs `lms-main-v1-rtl.scss`, etc.)